<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система управления культурным центром</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset>
         <normaloff>../../resourses/desktop.png</normaloff>../../resourses/desktop.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Рабочий стол</string>
       </attribute>
       <layout class="QHBoxLayout" name="desktopLayout"/>
      </widget>
      <widget class="QWidget" name="assignmentsTab">
       <attribute name="icon">
        <iconset>
         <normaloff>../../resourses/work.png</normaloff>../../resourses/work.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Заявки</string>
       </attribute>
       <layout class="QGridLayout" name="assignmentsLayout"/>
      </widget>
      <widget class="QWidget" name="eventsTab">
       <attribute name="icon">
        <iconset>
         <normaloff>../../resourses/event.png</normaloff>../../resourses/event.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Мероприятия</string>
       </attribute>
       <layout class="QGridLayout" name="eventsLayout"/>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="icon">
        <iconset>
         <normaloff>../../resourses/education.png</normaloff>../../resourses/education.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Образование</string>
       </attribute>
       <layout class="QVBoxLayout" name="educationLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Кружки</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout"/>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>Расписание</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Экспорт</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../../resourses/save.png</normaloff>../../resourses/save.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset>
         <normaloff>../../resourses/location.png</normaloff>../../resourses/location.png</iconset>
       </attribute>
       <attribute name="title">
        <string>&amp;Локации</string>
       </attribute>
       <layout class="QVBoxLayout" name="locationsLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
